tests/network/wireguard.pm

```diff
@@ -20,6 +20,21 @@ use utils;
 use lockapi;
 use mmapi 'wait_for_children';
 
+sub is_server {
+    return get_var('IS_MM_SERVER');
+}
+
+sub start_wgquick() {
+    # Start wg-quick with some debug output on failure
+    if (script_run('systemctl start wg-quick@wg0') != 0) {
+        script_run('systemctl status wg-quick@wg0');
+        save_screenshot();
+        script_run('journalctl -e');
+        save_screenshot();
+        die "wg-quick failed";
```

Comment:
Why this?

Reply:
This is to increase the verbosity in case of failure. `systemctl start wg0@qg-quick` is just failing without helpful error messages. This is to display what went wrong, in case the systemd service has some trouble.

I found this very useful while writing the test case, as it shows what went wrong and though we might still want to have it.

Reply:
Ah! The whole block `if (script_run('systemctl start wg-quick@wg0') != 0) { ... }` is happening only when the service failed! I saw `die` but didn't see the `if` condition :) All good then!